install/ansible/istio/vars/main.yml

```diff
@@ -14,4 +14,5 @@ istio_simple_samples:
   - name: sleep
     path: sleep/sleep.yaml
 
-istio_k8s_install_path: "install/kubernetes"
\ No newline at end of file
+istio_k8s_install_path: "install/kubernetes"
+kiali_version: v0.5.0
```

Comment:
why not keep this in `install_addons.yml`?

Reply:
I removed from that just to make all vars on the same place.
Should I return to `install_addons.yml`?

Reply:
I see and I'm ok to move this to same place. 

Same as https://github.com/istio/istio/pull/6985 , as master is frozen, can you please create PR against 1.0 branch?